docs/modules/operation/pages/geographical-map/searching.adoc

```diff
@@ -0,0 +1,73 @@
+## Searching
```

Comment:
```suggestion
# Map Search
```
Note that the heading of the file has to be level one, or it won't show up in the table of contents correctly. T